assets/src/components/cd/services/service/ServiceDetailsSidecar.tsx

```diff
@@ -92,6 +107,11 @@ export function ServiceDetailsSidecar({
             <Chip severity="success">None</Chip>
           )}
         </SidecarItem>
+        {helmRepository && (
+          <SidecarItem heading="Helm Repository">
```

Comment:
This should ideally include the helm icon like in the service table row

Reply:
I added the icons, I am just not sure if they are in the right place. I aligned them with the urls, is that ok?

Reply:
yeah they should be horizontally aligned w/ the urls